### PR TITLE
import ccn lite test cases

### PR DESCRIPTION
**Pull request by mehlis**

### CODE REVIEW

File: ccn-lite-client/Makefile [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+#### ../../RIOT 
+#### ../../boards   for board definitions (if you have one or more)
+#### 
+
+# name of your project
+export PROJECT=ccn
+
+# for easy switching of boards
+ifeq ($(strip $(BOARD)),)
+	export BOARD=msba2
```
Comment (OlegHahm) [inline ccn-lite-client/Makefile]: Should be `native` according to the howto.

File: ccn-lite-client/Makefile [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+endif
+
+# this has to be the absolute path of the RIOT-base dir
+#export RIOTBASE =$(CURDIR)/../../RIOT
+#export RIOTBOARD =$(CURDIR)/../../boards
```
Comment (OlegHahm) [inline ccn-lite-client/Makefile]: Remove the hashes.

**Comment by mehlis** [comment]
default BOARD to native and uncomment path to RIOT....done

**Comment by OlegHahm** [comment]
ACK and go.

**Pull request merged**